subscriptions/views/widget_poll.py

```diff
@@ -0,0 +1,21 @@
+from datetime import date
+
+from django.db.models import Q
+from django.http import JsonResponse
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from polls.models import Poll, Vote
+from polls.views import PollViewSet
+
+
+@api_view(["GET"])
+def widget_poll_view(request):
+    todays_poll = Poll.objects.filter(
+        Q(state="ACCEPTED") & Q(publication_date__lte=date.today())
```

Comment:
Why use `Q` here? `filter` is not enough?

Reply:
It's more explicit it's a `and` IMO

Reply:
I find it surprising but OK